src/include/btmem.h

```diff
@@ -894,7 +894,7 @@ struct __wt_page_deleted {
      * from memory rather than using the local variable, mark the shared transaction IDs volatile to
      * prevent unexpected repeated/reordered reads.
      */
-    volatile uint64_t txnid; /* Transaction ID */
+    wt_shared volatile uint64_t txnid; /* Transaction ID */
 
     wt_timestamp_t timestamp; /* Timestamps */
```

Comment:
why these are not shared?

Reply:
We built this list by looking for all concurrency primitives and reviewing the surrounding code, so it's possible some fields are missing. 
In the case of `PAGE_DELETED::durable_timestamp` I've re-reviewed and there are two point of interest:
- [`__tombstone_update_alloc`](https://github.com/wiredtiger/wiredtiger/blob/1befa2b41caad250425a84509c2464dd908f0ab6/src/btree/bt_delete.c#L437) where we could be missing a `WT_PUBLISH`, and
- [`__txn_apply_prepare_stage_page_del`](https://github.com/wiredtiger/wiredtiger/blob/1befa2b41caad250425a84509c2464dd908f0ab6/src/include/txn_inline.h#L186) where `prepare_state` is the synchronization variable and has been tagged as `wt_shared`

For this review we're trying to limit the variables tagged with `wt_shared` to only those closely coupled with a concurrency primitives, as otherwise we'd be tagging every almost every field visible to multiple threads and that's everything from the connection down.

All the same I'm happy to mark these as `wt_shared` if you thinks it's needed.

Reply:
I'm OK with that. But basically what we are doing is ignoring all the shared variables that can be read and written with relaxed ordering?

Reply:
Yep we're not tagging relaxed reads/writes. We want to track the fields where some level of ordering/inter-thread synchronization is required.